pkg/informers/informers.go

```diff
@@ -97,14 +101,8 @@ func Start() error {
 							continue
 						}
 
-						updatedBackup, err := veleroClient.Backups(backup.Namespace).Get(context.TODO(), backup.Name, metav1.GetOptions{})
```

Comment:
Is this tested in ci or was it tested manually? Are you sure it works without refreshing the resource? I assume it does because it is passing a pointer.

Reply:
good question, I think this will work but I'll give it a look

Reply:
The status does update in the UI, which I think is handled by this, so I would assume it's working